components/style/values/specified/basic_shape.rs

```diff
@@ -528,6 +592,19 @@ impl ToComputedValue for Polygon {
                                          .collect(),
         }
     }
+
+    #[inline]
+    fn to_specified_value(computed: &Self::ComputedValue, cx: &UncomputeContext) -> Self {
+        Polygon {
+            fill: ToComputedValue::to_specified_value(&computed.fill, cx),
+            coordinates: computed.coordinates.iter()
+                                         .map(|c| {
```

Comment:
Nit: I dunno if it's normal/preferred to line up the dots here like in `to_computed_value`...